content/blog/the-pulumi-intern-experience/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "The Pulumi Intern Experience"
+date: "2020-09-15"
+draft: true
+meta_desc: "A glimpse into my work and experiences at Pulumi as a summer intern"
+meta_image: meta.png
+authors: ["sashu-shankar"]
+tags: ["pulumi", "intern"]
+---
+
+What is the cloud? Three months ago, that one word simply meant a bunch of water suspended in the atmosphere, but now it means more than that.
+
+Hi, I’m [Sashu Shankar](https://www.linkedin.com/in/sashushankar/), a second-year computer science student at the University of Washington, and this is my life as a Pulumi intern!
+
+## My Pulumi Experience
+
+### Getting to Know the Team
+
+Let’s be real, onboarding with a new team in a remote situation was tough; the world was quite literally in a state of chaos, and each day was filled with talking to faces on the screen, but everyone was in the same situation, and that made it just a little more bearable. Coming into my first full software engineering internship, I was a nervous wreck, and I had millions of questions. What is the remote culture of the company going to be like? What is their standard for the quality and timeline of work, and am I going to be able to meet them? However, the team was always supportive, and it didn’t take long for me to feel like I was integrated with them and comfortable with the people around me. No matter the time of day, someone was always available to talk!
+
+### Asking Questions
+
+“There is no such thing as a dumb question” was a phrase I had been hearing a lot, but it never fully registered until recently. The “cloud” as we know it is quite vast. Setting it up takes hours, and fully understanding what is going on could take tenfold. Pulumi simplifies the process; however, there is still so much going on even when doing a single thing. This experience truly taught me that there really is no such thing as a dumb question and searching the internet to find answers is an expectation! Nobody knows all the answers, and that is completely okay.
+
+### Remote Lifestyle
+
+I’m definitely not a fan of the [remote environment](https://www.pulumi.com/blog/coronavirus-plan/), however, I’m the kind of person that is more productive at night, so it allowed me to shift my work to times when I knew I would get things done. That said, face to face interaction was something I missed a lot, as it’s understandably harder to get to know people over a computer. However, since Pulumi already has many remote engineers, it was a very smooth process to get set up and start work right away.
```

Comment:
```suggestion
I’m definitely not a fan of the [remote environment](https://www.pulumi.com/blog/coronavirus-plan/), however, I’m the kind of person that is more productive at night, so it allowed me to shift my work to times when I knew I would get things done. That said, face to face interaction was something I missed a lot, as it’s understandably harder to get to know people over a computer. However, since Pulumi already has many remote engineers, it was a very smooth process to get set up and start working right away.
```